grafana-plugin/src/utils/faro.ts

```diff
@@ -49,19 +50,25 @@ class BaseFaroHelper {
         url: url,
         isolate: true,
         instrumentations: [
-          new ErrorsInstrumentation(),
-          new WebVitalsInstrumentation(),
-          new ConsoleInstrumentation({
-            disabledLevels: [LogLevel.TRACE, LogLevel.ERROR],
+          ...getWebInstrumentations({
+            captureConsoleDisabledLevels: [LogLevel.TRACE, LogLevel.ERROR],
           }),
-          new SessionInstrumentation(),
         ],
         internalLoggerLevel: InternalLoggerLevel.VERBOSE,
-        session: (window as any).__PRELOADED_STATE__?.faro?.session,
         app: {
           name: appName,
           version: plugin?.version,
         },
+        sessionTracking: {
+          persistent: true,
+        },
+        beforeSend: (event) => {
+          if ((event.meta.page?.url ?? '').includes('grafana-oncall-app')) {
```

Comment:
Note: It's better to receive the string from the name in package.json but I didn't want to touch too many things.